app-frontend/src/app/pages/main/main.route.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+

```

Comment:
Since `login` is in `index.route.js` I think this should just be moved there too in order to be consistent _or_ create a `login.route.js`.

I think I am in favor of doing the first option until we get enough routes that it becomes cluttered.